javascript/i18n/phonenumbers/phonenumberutil_test.js

```diff
@@ -3550,6 +3550,151 @@ function testParseExtensions() {
       phoneUtil.parse('+1 (645) 123 1234 ext. 910#', RegionCode.US)));
 }
 
+function testParseHandlesLongExtensionsWithExplicitLabels() {
+  // Test lower and upper limits of extension lengths for each type of label.
+  /** @type {!i18n.phonenumbers.PhoneNumber} */
+  var nzNumber = new i18n.phonenumbers.PhoneNumber();
+  nzNumber.setCountryCode(64);
+  nzNumber.setNationalNumber(33316005);
+
+  // Firstly, when in RFC format: PhoneNumberUtil.CAPTURING_LONG_EXTN_DIGITS_WHEN_CLEAR
```

Comment:
These CAPTURING_LONG_EXTN_DIGITS_WHEN_CLEAR bits etc need updating to refer to vars that you actually declare

Reply:
Same in other test cases below.

Reply:
Oh yah. true. Some stale comments as I started JS some time back.
Corrected now.